### ISSUE TITLE
Script variant for italic

### ISSUE THREAD

**Issue by kag0** [JetBrains/JetBrainsMono#371]
I'm aware this one might not have mass appeal, but a variant of the italic font that uses a script (cursive) style (like [Dank Mono](https://gumroad.com/l/dank-mono)) would be an amazing option. Especially with script ligatures (like [Victor Mono](https://rubjo.github.io/victor-mono/)).

When italics are used for language keywords I think it has a subtle but appealing effect that makes it quicker to identify keywords compared to regular italics.
![](https://i.imgur.com/JAEezpb.png)
See also [Operator](https://www.typography.com/blog/introducing-operator)

**Comment by CodeCrazy-ywt** [comment]
Well, I'm afraid that it's against the original philosophy of Jetbrains Mono -- to reduce the noise of the font and improve the readability of the code. As for me, I need to stop for a while to recognize letters, especially b, f, s.

**Comment by Omnieboer** [comment]
I agree with @CodeCrazy-ywt that this wouldn't necessarily fit right now, but if this could be added if (when?) variant fonts are supported by JetBrains Mono and by a lot more editors, this would be a really cool feature. I like the extra layer of distinction one can make with cursive. Keywords are now individually styled in JetBrains IDEs for me, but this does look amazing to me.

**Comment by philippnurullin** [comment]
Hi @kag0 Thanks for suggestion. We don't have plans to adopt this letter constructions. Too much uncommon forms is wasted time on recognition of the letters. It's pretty but it may affect performance in case of JetBrains Mono.

**Comment by victor-falcon** [comment]
This is the only reason why I'm using Fira Code instead of JetBrains Mono 😅

**Comment by mfocko** [comment]
@victor-falcon I'm not sure if I follow you… Fira Code doesn't even have »dedicated« italics, the font just gets „italicized“ by an algorithm for italics

**Comment by victor-falcon** [comment]
@mfocko I meant Cascadia, sorry
